Shared/qml/About.qml

```diff
@@ -105,7 +105,7 @@ Item {
 
             Text {
                 anchors.horizontalCenter: parent.horizontalCenter
-                text: "Built with ArcGIS Runtime SDK 100.9 for Qt"
+                text: "Built with ArcGIS Runtime SDK 200.3 for Qt"
```

Comment:
This seems off and we may be updating it again.

Reply:
Removed the specific version number for now and replaced 'Runtime' with 'Maps SDK'. Will make a sweeping effort in an upcoming branch to update all version numbers and references to Runtime to be consistent.